indico/modules/events/papers/schemas.py

```diff
@@ -80,28 +102,64 @@ class PaperRevisionSchema(mm.ModelSchema):
     state = EnumField(PaperRevisionState)
     timeline = PaperRevisionTimelineField()
     judgment_comment_html = Function(lambda revision: escape(revision.judgment_comment))
+    reviewer_data = Method('_get_reviewer_data')
 
     class Meta:
         model = PaperRevision
         fields = ('id', 'submitted_dt', 'judgment_dt', 'submitter', 'judge', 'spotlight_file', 'files',
-                  'is_last_revision', 'number', 'state', 'timeline', 'judgment_comment', 'judgment_comment_html')
+                  'is_last_revision', 'number', 'state', 'timeline', 'judgment_comment', 'judgment_comment_html',
+                  'reviewer_data')
+
+    def _get_reviewer_data(self, revision):
+        if not revision.is_last_revision:
+            return None
+
+        data = dict(revision.get_reviewer_render_data(self.context.get('user')))
+        review_type_schema = PaperReviewTypeSchema()
+        for name in {'groups', 'missing_groups', 'reviewed_groups'}:
```

Comment:
`()` instead of `{}`? (no benefit of using a set and AFAIK there are more efficient opcodes for creating a small tuple - not that it really matters here...)

Reply:
Yeah I can change it so it is more consistent since we use tuples pretty much everywhere.